4-Deployment/README.md

```diff
@@ -0,0 +1,206 @@
+# Deploy your Angular application to Azure Cloud and use Azure services to manage your operations
+
+ 1. [Overview](#overview)
+ 1. [Scenario](#scenario)
+ 1. [Prerequisites](#prerequisites)
+ 1. [Registration](#registration)
+ 1. [Deployment](#deployment)
+ 1. [Explore the sample](#explore-the-sample)
+ 1. [More information](#more-information)
+ 1. [Community Help and Support](#community-help-and-support)
+ 1. [Contributing](#contributing)
+ 1. [Code of Conduct](#code-of-conduct)
+
+## Overview
+
+This sample demonstrates how to deploy an Angular single-page application (SPA) coupled with a .NET Core web API to **Azure Cloud** using [Azure Storage](https://docs.microsoft.com/azure/storage/blobs/) and [Azure App Service](https://docs.microsoft.com/azure/app-service/), respectively. To do so, we will use the [same code sample from Chapter 3](../3-Authorization-II/1-call-api).
+
+> :information_source: You can choose to deploy the [B2C sample from Chapter 3](../3-Authorization-II/1-call-api-b2c) instead. Steps below are more or less the same. 
```

Comment:
**B2C sample from Chapter 3** Link is broken. It's **3-Authorization-II/2-call-api-b2c**.